RELEASE-NOTES.md

```diff
@@ -31,6 +31,7 @@
   * Currently only supports SQL scripts: `post_deploy: [{sql_script: script.sql}]`
 * Added `snow spcs service execute-job` command, which supports creating and executing a job service in the current schema.
 * Added `snow app events` command to fetch logs and traces from local and customer app installations
+* Added support for `SNOWFLAKE_CLI_RESOURCE_SUFFIX` environment variable to be used as a suffix for Native App package and app names
```

Comment:
not sure what's the common practice on this. Since this is purely internal for testing purposes, should we skip it from release notes?

Reply:
i added it cause it could technically be used by end-users. I'm ok leaving it off if we want to consider this an undocumented feature

Reply:
I think the variable is rather misleading since it seems to imply it will cause all resources to be suffixed, but that's not the case. I'd leave this to be undocumented since we're not guaranteeing a user can build workflows on it.

Reply:
`SNOWFLAKE_CLI_TEST_RESOURCE_SUFFIX` might convey the intention here a little more clearly? We are not in any way saying this should be used in a production setting.